aqt/archives.py

```diff
@@ -72,7 +73,7 @@ def __init__(self, version, os_name, target, base, timeout=(5, 5)):
         self.os_name = os_name
         self.target = target
         self.archives = []
-        self.base = base
+        self.base = posixpath.join(self.base, "online", "qtsdkrepository")
```

Comment:
This change causes the parameter `base` to be ignored. Was this line meant to be:

```python
self.base = posixpath.join(base, "online", "qtsdkrepository")
```

When I try to run `aqt list  5.15.2 linux desktop` after this change, I get a stack trace ending in:

```bash
  File "D:\Documents\PycharmProjects\aqtinstall\aqt\archives.py", line 76, in __init__
    self.base = posixpath.join(self.base, "online", "qtsdkrepository")
AttributeError: 'PackagesList' object has no attribute 'base'
```

Reply:
You're absolutely right, sorry!

Reply:
https://github.com/miurahr/aqtinstall/pull/286/